pkg/models/mto_category.go

```diff
@@ -22,6 +22,12 @@ type MTOCategory struct {
 	ParentID *uuid.UUID `json:"parent_id" db:"parent_id"`
 }
 
```

Comment:
Just a comment on this type might be helpful to explain what it's for, contextually!

```suggestion
// MTOCategories is a struct meant to represent the Category and Subcategory associated with a given MTO Milestone.
// This struct exists to allow us to resolve/fetch both Category and Subcategory at the same time and return them together in a single resolver.
```